strings/minion.py

```diff
@@ -0,0 +1,66 @@
+# The Minion Game
+"""Task
+Your task is to determine the winner of the game and their score.
+
+Kevin and Stuart want to play the 'The Minion Game'.
+
+Game Rules:
+Both players are given the same string, S.
+Both players have to make substrings using the letters of the string S.
+Stuart has to make words starting with consonants.
+Kevin has to make words starting with vowels.
+The game ends when both players have made all possible substrings.
+
+Scoring
+A player gets +1 point for each occurrence of the substring in the string S.
+
+For Example:
+String S = BANANA
+Kevin's vowel beginning word = ANA
+Here, ANA occurs twice in BANANA. Hence, Kevin will get 2 Points.
+
+Input Format:
+A single line of input containing the string S.
+Note: The string S will contain only uppercase letters: [A - Z].
+
+Constraints:
+0 < len(S) <= 10^6
+
+Output Format:
+Print one line: the name of the winner and their score separated by a space.
+If the game is a draw, print Draw.
+
+Note :
+Vowels are only defined as AEIOU. In this problem, Y is not considered a vowel.
+"""
+
+
+def minion_game(string):
+    vowels = 'AEIOU'
+    kevin = sum(len(string) - i for i in range(len(string)) if string[i] in vowels)
+    stuart = sum(len(string) - i for i in range(len(string)) if string[i] not in vowels)
```

Comment:
I think here's the wrong winner condition implemented.
Expected condition for Kevin: total amount of words which start with a vowel and are substrings of `string`.
Implemented condition for Kevin: summary of (some) substring lenghts which starts with a vowel.
E.g. for `BANANA`: 
expected list for Kevin is [`A`, `A`, `A`, `AN`, `AN`, `ANA`, `ANA`, `ANAN`, `ANANA`] and score is 9
implemented list for Kevin is [`ANANA`, `ANA`, `A`] and score is 9
It seems the results are the same at least for the hacker-rank test cases but I have a feeling something's wrong...